Plutarch/Internal.hs

```diff
@@ -105,7 +107,7 @@ mapTerm f (TermResult t d) = TermResult (f t) d
 mkTermRes :: RawTerm -> TermResult
 mkTermRes r = TermResult r []
 
--- | Type of `s`.
+-- | Type of `s` in `Term s a`. See: "What is the `s`?" section on the Plutarch guide.
```

Comment:
on -> in
Do we have a stable link for the Plutarch guide?

Reply:
I suppose we should consider the one on `master` stable until we set up a website